js/main.js

```diff
@@ -69,7 +69,7 @@
             clearChildren(username);
             var a = document.createElement("a");
             a.setAttribute("target", "blank");
-            a.href = "https://github.com/" + user;
+            a.href = "https://github.com/" + (user.startsWith("@") ? user.slice(1) : user);
```

Comment:
I see.

However, then please use `user.charAt(0) === "@"` instead of using `startsWith` which is an experimental thing.